js/components/portal-dashboard/header.tsx

```diff
@@ -2,18 +2,25 @@ import React from "react";
 
 import ccLogoSrc from "../../../img/cc-logo.png";
 import css from "../../../css/portal-dashboard/header.less";
```

Comment:
I adopted Kirk's rule of always putting the styling import last in the list of imports.